CHANGES.rst

```diff
@@ -31,6 +33,8 @@ Changelog
 - Explicitly set required=False to all "Bool" zope.schema fields in the ISolrSchema interface. zope.schema does not fall back to default=False by default (see https://github.com/zopefoundation/zope.schema/blob/master/CHANGES.rst#611-2021-10-13)
   [tisto]
 
+- Add official support for Python 3.10
+  [tisto]
```

Comment:
This is added twice :)

Reply:
@gforcada good catch. Thanks! No clue why I added this in the 9.0.0 release already...

Reply:
Will fix this after the merge before making a release...